backend-rust/src/migrations/m0025_fix_passive_delegator_stake.rs

```diff
@@ -0,0 +1,64 @@
+//! Migration reindexing the current stake for delegators to the passive pool.
+//!
+//! The stakes for these delegators got offset by a bug in migration 0013, which
+//! did not update the passive delegators properly. This was fixed again in
+//! migration m0022, but the passive delegators with restake_earnings enabled
+//! missed their increase in stake due to this.
+use super::{SchemaVersion, Transaction};
+use anyhow::Context;
+use concordium_rust_sdk::v2::{self, BlockIdentifier};
+use futures::{StreamExt, TryStreamExt};
+
+/// Resulting database schema version from running this migration.
+const NEXT_SCHEMA_VERSION: SchemaVersion = SchemaVersion::FixPassiveDelegatorsStake;
+
+/// Migration reindexing the current stake for delegators to the passive pool.
+pub async fn run(
+    tx: &mut Transaction,
+    endpoints: &[v2::Endpoint],
+) -> anyhow::Result<SchemaVersion> {
+    let latest_height: Option<i64> =
+        sqlx::query_scalar("SELECT height FROM blocks ORDER BY height DESC LIMIT 1")
+            .fetch_optional(tx.as_mut())
+            .await?;
+    let Some(height) = latest_height else {
+        return Ok(NEXT_SCHEMA_VERSION);
+    };
+    let block = BlockIdentifier::AbsoluteHeight(u64::try_from(height)?.into());
```

Comment:
Suggestion would be to propagate the `latest` so it is clear what block we look up.

```suggestion
    let latest_block = BlockIdentifier::AbsoluteHeight(u64::try_from(latest_height)?.into());
```